samples/kvsWebRTCClientViewer.c

```diff
@@ -102,6 +102,25 @@ INT32 main(INT32 argc, CHAR *argv[])
         goto CleanUp;
     }
 
+    if (!pSampleConfiguration->trickleIce) {
+        printf("[KVS Viewer] Non trickle ice. Wait for Candidate collection to complete\n");
+        MUTEX_LOCK(pSampleConfiguration->sampleConfigurationObjLock);
+        locked = TRUE;
+
+        while (!ATOMIC_LOAD_BOOL(&pSampleStreamingSession->candidateGatheringDone)) {
+            CHK_WARN(!ATOMIC_LOAD_BOOL(&pSampleStreamingSession->terminateFlag), STATUS_INTERNAL_ERROR, "application terminated and candidate gathering still not done");
```

Comment:
Why internal error? Perhaps operation failed or another error.

Reply:
the closest status I found is STATUS_OPERATION_TIMED_OUT

Reply:
STATUS_INVALID_OPERATION might do too. TIme out also OK in my opinion.